snapcraft/internal/lxd/_project.py

```diff
@@ -81,7 +82,8 @@ def _configure_container(self):
             subprocess.check_call([
                 'lxc', 'config', 'set', self._container_name,
                 'raw.idmap',
-                'both {} 0'.format(os.getenv('SUDO_UID', os.getuid()))])
+                'both {} {}'.format(os.getenv('SUDO_UID', os.getuid()),
+                                    os.getuid())])
```

Comment:
Will this be thrown off if we're running with sudo (given the use of uid as opposed to euid)?

Reply:
Afair it only affects mounting folders as LXD devices. And `_user` would be set to `'root'` in the sudo case.